### ISSUE TITLE
Dune Analytics Queries for NFT Audit

### ISSUE THREAD

**Issue by thewanderingeditor** [hoprnet/hopr-devrel#204]
# Deadline

April 5th (as this is a prerequisite for much of #207), and the NFT program audit should be completed before the sprint beginning April 11th

# Context

As we move into Staking Season 3, we need to conduct an analysis and cost audit of the NFT program, which was originally conceived as a short-term stopgap before the launch of cover traffic rewards. This will primarily be conducted by @thewanderingeditor, but some new Dune Analytics queries are needed to get the full picture.

We need to assess:

1) The overall cost of the staking program
2) The overall cost of the NFTs by season
3) The relative costs of particular *types* of NFTs
4) The NFTs held by the largest stakers, and how and when they acquired them

This will allow us to define the budget for issuing new NFTs and determine whether certain NFTs should be discontinued. T

We currently have three relevant dashboards on Dune:

- [HOPR Staking Program [Season 1]](https://dune.xyz/hoprnet/HOPR-Staking-Program)
- [HOPR Staking Program [Season 2]](https://dune.xyz/hoprnet/HOPR-Staking-Program-Season-2)
- [HOPR NFT](https://dune.xyz/hoprnet/HOPR-NFT)

These are sufficient for 1) and 4) (with some manual inspection of on-chain data), but not 2) or 3).

# Suggested Queries

For a thorough audit, the HOPR NFT dashboard will need to be extended. 

- [ ] We should begin by discussing the most efficient way to get the information needed, as it's not intuitive and some queries could end up being too complex. 

Some suggested queries, for each NFT type:

- [ ]  The number of each type held versus staked (similar to [this query](https://dune.xyz/queries/268433/504557), which is by address). If possible, it might be useful to add the number transferred to this (i.e., which move to an address other than the initially issued one) as a proxy for them being resold.
- [ ] Their relative contribution of each type to the overall cost of the NFT program (very roughly estimated as the actual cost minus the theoretical cost if no HOPR NFTs existed and all payouts were at the base rate of 18.25%)

**Comment by thewanderingeditor** [comment]
It was decided that we don't have sufficient resources or need to conduct a full NFT audit. The dune queries we have are sufficient for our new plan. Will close this issue and make a new one reflecting the modified audit.

**Comment by thewanderingeditor** [comment]
NFT Cost Audit issue was created here: https://github.com/hoprnet/hopr-devrel/issues/213
